### PR TITLE
Block post base

### LINKED ISSUE #39: Post Blocks base
Base/default (used on homepage and archive pages)

### PR DESCRIPTION
**Pull request by jdromero88**
closes #39

### CODE REVIEW

File: wp-content/themes/reconasia/template-parts/block-post.php [hunk 1]
```diff
@@ -12,16 +12,26 @@
...
 
 ?>
-
 <article <?php post_class('post-block post-block--post'); ?> id="post-<?php the_ID(); ?>">
+	<div class="post-block-right">
+		<?php
+			// if ( !is_front_page() ){
+				// echo get_the_post_thumbnail( get_the_ID(), 'large' );		
+			// }
+			echo get_the_post_thumbnail();
```
Comment (urchykli) [inline wp-content/themes/reconasia/template-parts/block-post.php:21]: Will you please make the thumbnail a link to the post and have the hover effect be applied to the title when you hover over the image?

File: wp-content/themes/reconasia/assets/_scss/components/_post-block.scss [hunk 2]
```diff
@@ -1,22 +1,31 @@
...
 .post-block {
-  &__title {
-    color: $color-black-100;
+  margin-top: rem(40);
 
-    a {
-      &:visited {
-        // color: $color-text-gray-70;
-      }
+  &__title {
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/components/_post-block.scss:6]: Don't forget the hover effect.  The styles are at the top of the components page in a box called Post Title Link.

File: wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss [hunk 3]
```diff
@@ -29,17 +29,22 @@
...
+    color: $color-black-165;
+    @extend %text-style-label-small-strong;
+  }
 
-    &::before {
-      content: '|';
-      margin-right: rem(8);
-      color: var(--post-meta-divider);
-    }
+  &__categories {
```
Comment (urchykli) [inline wp-content/themes/reconasia/assets/_scss/components/_post-meta.scss:37]: Let's ask Christina in the Recon Asia slack channel if this should have a hover state.

File: wp-content/themes/reconasia/template-parts/block-post.php [hunk 4]
```diff
@@ -12,16 +12,26 @@
...
-	reconasia_authors();
-	?>
+	<div class="post-block-left">
+		<?php
+			the_title( '<h3 class="post-block__title"><a href="' . esc_url( get_permalink() ) . '">', '</a></h3>' );
+			
+			the_excerpt();
+			
+			reconasia_authors();
+			reconasia_posted_on();
```
Comment (urchykli) [inline wp-content/themes/reconasia/template-parts/block-post.php:32]: Let's use the [WP date format](https://wordpress.org/support/article/formatting-date-and-time/) to format the date correctly.  The function to call the posted on date accepts the format string, so you can do something like `reconasia_posted_on('M j, Y');`

**Review by urchykli — CHANGES_REQUESTED** [review]
Nice first pass on this, Jose!  I left some inline comments, but let's also do the following.

- Delete the file `wp-content/themes/reconasia/assets/_scss/blocks/issues/_reconasia-original.scss` since we don't need it.
- In `template-tags.php` on line 92 let's adjust the links so the word "and" is used before the last author when there are multiple authors.
- In `_post-meta.scss` let's remove `color: var(--post-meta);` on line 4, since we aren't using it.
- In `front-page.php`, we need two sections, one for recent posts and one for featured posts.  You don't need to handle the functionality, but we should add the other section, so you can style the post blocks based on the correct section.

**Comment by jnschrag** [comment]
Closing this pull request because @jdromero88 is working on his changes on a new branch because of a broken rebase.

**Pull request closed without merging**